packages/ui/src/kit/VDateInput/VDateInput.vue

```diff
@@ -0,0 +1,228 @@
+          <td class="v-date-input-day">W</td>
+          <td class="v-date-input-day">T</td>
+          <td class="v-date-input-day">F</td>
+          <td class="v-date-input-day">S</td>
+        </tr>
+        <tr v-for="(week, i) in currentCalendar" :key="`${innerYear}-${i}`">
+          <td v-for="day in week" :key="day.date.toString()">
+            <button
+              :class="{
+                'v-date-input-day': true,
+                'v-date-input-control': true,
+                'is-today': day.isToday,
+                'is-disabled': !day.inTargetMonth,
+                'is-selected': isSelectedDay(day),
+              }"
+              :disabled="
+                !day.inTargetMonth || day.isToday || isSelectedDay(day)
+              "
+              @click="onClickDay(day)"
+            >
+              {{ day.origin.date() }}
+            </button>
+          </td>
+        </tr>
+      </table>
+    </div>
+  </div>
+</template>
+
+<script>
+import dayjs from 'dayjs';
+import { getMonthCalendar } from '@/utils/date';
+import ThemeMixin from '@/mixins/ThemeMixin';
+import VSvgIcon from '@/kit/VSvgIcon';
+import InputAtom from '@/atom/input-atom/input-atom';
+
+export default {
+  name: 'VDateInput',
+
+  props: {
+    value: {
+      type: [String, Date],
+      default: '',
+    },
+  },
+
+  data: () => ({
+    innerMonth: 0,
+    innerYear: 2000,
+    isCalendarVisible: false,
+  }),
+
+  computed: {
+    vDateInputCssClass() {
+      return Object.assign({}, this.themeCssClass, {
+        'v-date-input': true,
+      });
+    },
+
+    date() {
+      return dayjs(this.value);
+    },
+
+    formattedInnerMonth() {
+      return dayjs()
+        .month(this.innerMonth)
+        .format('MMMM');
+    },
+
+    currentCalendar() {
+      return getMonthCalendar(dayjs(new Date(this.innerYear, this.innerMonth)));
+    },
+
+    formattedValue() {
+      return this.date.format('YYYY/MM/DD');
+    },
+  },
+
+  watch: {
+    isCalendarVisible(val) {
+      if (val) {
+        document.body.addEventListener('click', this.onClickAnywhere);
+        window.addEventListener('keypress', this.onWindowEscPress);
+      } else {
+        document.body.removeEventListener('click', this.onClickAnywhere);
+        window.removeEventListener('keypress', this.onWindowEscPress);
+      }
+    },
+
+    value(val, oldVal) {
+      const oldDate = dayjs(oldVal);
+      const newDate = dayjs(val);
+      const newDateMonth = newDate.month();
+      const newDateYear = newDate.year();
+
+      if (newDateMonth !== oldDate.month() || newDateYear !== oldDate.year()) {
+        this.innerMonth = newDateMonth;
+        this.innerYear = newDateYear;
+      }
+    },
+  },
+
+  methods: {
+    isSelectedDay({ origin }) {
+      return (
+        this.date.date() === origin.date() &&
+        this.date.month() === origin.month() &&
+        this.date.year() === origin.year()
+      );
+    },
+
+    onBlurInput(e) {
+      console.log(e);
```

Comment:
`console.log` artifact, 
and can we not bind to blur/focus actions, but use css:hover?